sdcm/cluster.py

```diff
@@ -1839,6 +1840,15 @@ def config_setup(self,
 
             if ldap:
                 scylla_yml.update(self.get_ldap_config())
+            elif ms_ad_ldap:
+                scylla_yml.update({
+                    'ldap_attr_role': 'cn',
+                    'ldap_bind_dn': 'cn=admin,dc=scylla-qa,dc=com',
+                    'ldap_bind_passwd': 'scylla-qa2021!',
```

Comment:
It's unsafe to expose the plaintext password to public git repo.
Can you save it to qa-keystore in S3?

Reply:
added few information here to `KeyStore`